cmake/modules/SearchInstalledSoftware.cmake

```diff
@@ -1115,6 +1115,18 @@ if (jemalloc)
   endif()
 endif()
 
+#---Check for liburing----------------------------------------------------------------
```

Comment:
Can you add please fail-on-missing in account as well? Here is a nice example:
https://github.com/root-project/root/blob/2d2bc4ef56f68b707a0b918b1087241b4ffcf7e1/cmake/modules/SearchInstalledSoftware.cmake#L561-L572

Reply:
Thank you, done in 6f1efa8967efe0071847f2d5abe0681219f7291f